llvm/docs/AMDGPUUsage.rst

```diff
@@ -5468,9 +5468,13 @@ If the *Target Properties* column of :ref:`amdgpu-processor-table` specifies
 Instead the flat SCRATCH instructions are used.
 
 Otherwise, Private Segment Buffer SGPR register is used to initialize 4 SGPRs
-that are used as a V# to access scratch. CP uses the value provided by the
-runtime. It is used, together with Scratch Wavefront Offset as an offset, to
-access the private memory space using a segment address. See
+that are used as a V# to access scratch. 
+FIXME: The compiler synthetizes the initialization value for the Private Segment
```

Comment:
Not sure why this is "FIXME". Also typo 'synthetizes'

Reply:
removed FIXME and corrected the typo